cool-core/src/test/java/com/nus/cool/core/model/CoolModelTest.java

```diff
@@ -6,8 +6,12 @@
 import com.nus.cool.core.cohort.funnel.FunnelQuery;
 import com.nus.cool.core.iceberg.query.IcebergQuery;
 import com.nus.cool.core.iceberg.result.BaseResult;
+import com.nus.cool.core.io.readstore.ChunkRS;

```

Comment:
These import are not used, shall remove them

Reply:
these imports are used in this test file, but we will move these test cases into other classes. We will clean it in another branch.